### PR TITLE
fix Reclaim your privacy link

### PR DESCRIPTION
**Pull request by MarnixCroes**
current link gives 404

### CODE REVIEW

**Comment by MarnixCroes** [comment]
@CAnorbo any idea what broke it?
the recent updates of dependencies?

**Review by CAnorbo — CHANGES_REQUESTED** [review]
This is not good.
Should look like this:

actions:
  - text: Reclaim Your Privacy →
    link: /getting-started/
    type: primary

File: docs/README.md [hunk 1]
```diff
@@ -4,10 +4,8 @@ heroText: Wasabi Documentation
 heroImage: /Logo_without_text_dark.png
 heroImageDark: /Logo_without_text.png
 tagline: An archive of knowledge about Wasabi Wallet, an open-source, non-custodial and privacy-focused Bitcoin wallet for desktop.
-actions:
-  - text: Reclaim Your Privacy →
-    link: /getting-started
```
Comment (CAnorbo) [inline docs/README.md:9]: Just add the / sign at the end of the link line.
Don't change the names.

Comment (MarnixCroes) [reply docs/README.md:9]: done

**Comment by CAnorbo** [comment]
tACK

**Review by CAnorbo — APPROVED** [review]

**Review by yahiheb — APPROVED** [review]
tACK

**Pull request merged**